src/components-v1/repository/AnalyzeTextDrawer.vue

```diff
@@ -195,5 +204,18 @@ export default {
   &-content {
     padding: 4px 16px 8px;
   }
+
+  &--slide-enter-active, &--slide-leave-active {
+    transition: margin-bottom .2s ease-out;
+    overflow: hidden;
+  }
+
+  &--slide-enter, &--slide-leave-to {
+    margin-bottom: -200px;
```

Comment:
you tray use -100% ?

Reply:
Yes, with percent is better than px, thanks!